index.html

```diff
@@ -32,6 +32,16 @@ <h1>I'm Steph, a Junior Software Developer</h1>
 	</section>
 </header>
```

Comment:
this also looks like a stray closing tag. I can't find an opening tag for header (although can see 'head' further up)